File: templates/Uwp/_composition/_shared/Feature.UnitTests.Core.SampleData.Chart.NUnit._VB/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.vb
'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public Class Tests
    '^^
    '{[{

    ' TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Test>
    Public Sub EnsureSampleDataServiceReturnsChartData()
        Dim actual = SampleDataService.GetChartSampleData()

        Assert.IsNotEmpty(actual)
    End Sub
    '}]}
End Class

File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.ContentGrid.MSTest/Param_ProjectName.Core.Tests.MSTest/Tests_postaction.cs
//{[{
using System.Threading.Tasks;
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.MSTest
{
    public class Tests
    {
        //^^
        //{[{

        // TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [TestMethod]
        public async void EnsureSampleDataServiceReturnsContentGridDataAsync()
        {
            var actual = await SampleDataService.GetContentGridDataAsync();

            Assert.AreNotEqual(0, actual.Count());
        }
        //}]}
    }
}
